src/layout/header/Login.tsx

```diff
@@ -39,7 +39,7 @@ const Login: FC = () => {
     if (USE_MOCK_DATA) {
       dispatch(setUser(mockUser));
     } else {
-      Auth.federatedSignIn();
+      Auth.federatedSignIn({ customProvider: FEIDE_IDENTITY_PROVIDER });
```

Comment:
hvilken betydning har dette?

Reply:
Slipper den mellomskjermen som viser `Sign in with your corporate ID`. Går rett til Feide og input-felt for brukernavn/passord

Reply:
nice!